### PR TITLE
Development

### PR DESCRIPTION
**Pull request by PolinaStamenova**
### А mobile web application to check a list of metrics (numeric values).

- Fetching data from API using axios library. 
- Create logic to display cities in Spain on the home page with temperature.
- Create logic to display single city with some more metrics.
- Use Link, Route and Router to navigate.
- Create store where all the fetch data from the API is stored.
- Create test for the API.

_Thank you for taking time to review my PR and sorry for the delay!_  😄 🌵

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,66 @@
...
+## Getting Started
 
-### `npm run eject`
+To get a local copy up and running follow these simple example steps.
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Instructions
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+$ cd <folder>
```
Comment (arslanbisharat) [inline README.md:31]: - [ ] [Optional] It would be great if this is in markdown using ` ` .

File: README.md [hunk 2]
```diff
@@ -1,70 +1,66 @@
...
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+$ git clone git@github.com:PolinaStamenova/wetrics-webapp.git
```
Comment (arslanbisharat) [inline README.md:33]: - [ ] [Optional] It would be great if this is in markdown using ` ` .

File: README.md [hunk 3]
```diff
@@ -1,70 +1,66 @@
...
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## Acknowledgments
 
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+- Microverse
+- Creative Commons license of the design
```
Comment (arslanbisharat) [inline README.md:66]: - [ ] [Optional] Thank you for giving credit to `Creative Commons` but it would be great if you give to the original designer.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: README.md (hunk 1 above)
Comment (PolinaStamenova) [reply README.md:31]: Thank you!

I will fix it.

File: README.md (hunk 3 above)
Comment (PolinaStamenova) [reply README.md:66]: Okay I will change this as well. 😄

**Pull request merged**